post-deployment.yml

```diff
@@ -54,6 +54,14 @@
       ansible.builtin.import_role:
         name: run_hook
 
+      # Gemini 2.5 Flash Prompt: what gather_facts I can run to collect
```

Comment:
oh really this comment is needed :D ?

Reply:
Let me follow the openstack convention here: https://openinfra.org/legal/ai-policy and Add generated by and prompt in the commit message itself

Reply:
@danpawlik @evallesp Updated the commit message, let me know your thoughts on this.